`'async' modifier cannot appear on a type member.` error in `IDatabase.ts`

```
> tsc -p tsconfig.json

src/databases/IDatabase.ts(2,5): error TS1070: 'async' modifier cannot appear on a type member.
```

i believe this is a recent change in typescript 4.0. because the `async` modifier is useless in interfaces and abstract methods, it's no longer allowed

--- a/src/databases/IDatabase.ts
+++ b/src/databases/IDatabase.ts
@@ -1,5 +1,5 @@
 export interface IDatabase {
-    async init(): Promise<void>;
+    init(): Promise<void>;
 
     prepare(type: QueryType, query: string, params?: any[]): Promise<any | any[] | void>;
 }

